packer/packer.go

```diff
@@ -0,0 +1,132 @@
+package packer
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/jfrog/terraform-provider-shared/predicate"
+	"github.com/jfrog/terraform-provider-shared/util"
+	"reflect"
```

Comment:
I know `gofmt` doesn't care about the ordering here. May be not even `goimports` but we are pretty consistent ourselves putting 3rd party modules last.

Reply:
Oh, I just let goland do its thing. I didn't realize this was important